src/engines/html5/media-source/adapters/fairplay-drm-handler.js

```diff
@@ -34,7 +34,7 @@ class FairplayDrmHandler {
       responseFilter: (type, response) => {
         let responseObj = {};
         try {
-          const dataView = new DataView(response.data);
+          const dataView = new DataView(response.data.trim());
```

Comment:
trim should be on the text output not the binary.

Reply:
trim wouldn't delete empty chars from binary?